prow/cmd/plank/main_test.go

```diff
@@ -0,0 +1,136 @@
+/*
+Copyright 2017 The Kubernetes Authors.
```

Comment:
Probably should be 2019

Reply:
I think that's what it added by default.  Not sure where the default config is for template comment, but perhaps need to change that.